src/core/Modal/styles.ts

```diff
@@ -0,0 +1,36 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: () => ({
+    height: '80%',
+    width: '70%',
+    margin: '2rem auto',
+    color: theme.palette.text.primary,
+    background: theme.palette.background.paper,
+    borderRadius: 3,
+    textAlign: 'center',
+    outline: 'none',
+    overflowX: 'hidden',
+    overflowY: 'auto',
+    justifyContent: 'center',
+    alignItems: 'center',
+  }),
+
+  closeButton: () => ({
+    fontSize: '1rem',
+    fontWeight: 1000,
+    display: 'block',
```

Comment:
Display is block by default so we can omit this, if not otherwise.

Reply:
removing the block won't work, if removed the close Button will move to the center as in the root class its mention align-items to be in the center.